.gitlab-ci.yml

```diff
@@ -16,6 +16,7 @@ variables:
   SSTATE_LOCATION: "${RUNNER_SSTATE}/${CI_PROJECT_NAME}"
   SSTATE_MIRRORS: "file://.* ${SSTATE_LOCATION}/PATH;downloadfilename=PATH"
   SSTATE_MAX_AGE: "3d"
+  DISTRO_APT_PREMIRRORS: 'http://deb\.debian\.org https://debian.inf.tu-dresden.de\n'
```

Comment:
Why? I know reasons, but they should go into the commit message.

Reply:
Added the reason.